tests/integration/test_asynchronous_metric_jemalloc_profile_active/test.py

```diff
@@ -0,0 +1,74 @@
+import time
+
+import pytest
+from helpers.cluster import ClickHouseCluster
+
+cluster = ClickHouseCluster(__file__)
+node1 = cluster.add_instance(
+    "node1",
+    main_configs=["configs/asynchronous_metrics_update_period_s.xml"],
+    env_variables={"MALLOC_CONF": "background_thread:true,prof:true"},
+)
+
+
+@pytest.fixture(scope="module")
+def started_cluster():
+    try:
+        cluster.start()
+        yield cluster
+
+    finally:
+        cluster.shutdown()
+
+
+# Tests that the system.asynchronous_metric_log table gets populated.
+# asynchronous metrics are updated once every 60s by default. To make the test run faster, the setting
+# asynchronous_metric_update_period_s is being set to 2s so that the metrics are populated faster and
+# are available for querying during the test.
+def test_event_time_microseconds_field(started_cluster):
```

Comment:
I just noticed that the test is named wrong

Reply:
Oh, my fault. You mean the name of the test and the function name in test.py should be the same. Sorry.

Reply:
It doesnt have to be same, I just dont understand why it's named like that
You dont test microsecond field